custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -31,6 +31,7 @@ def validate_phone_number(string_value):
     if string_value is None or string_value.strip() in ['', '0']:
         return ''
     else:
+        assert int(string_value) > 0
         phone_number = str(int(string_value))
```

Comment:
@nickpell I'm not sure it's a good idea to coerce to an int, since that might do things like strip valid leading zeroes.  Python strings have a builtin `isdigit` method which you could use:
```python
In [1]: "abc".isdigit()
Out[1]: False

In [2]: "123".isdigit()
Out[2]: True

In [3]: "123abc".isdigit()
Out[3]: False
```
Or you could just call `int(string_value)` to see if it fails and do nothing with the result.